firefox-ios/WidgetKit/OpenTabs/OpenTabsWidget.swift

```diff
@@ -57,6 +60,29 @@ struct OpenTabsView: View {
                         .lineLimit(1)
                         .font(.system(size: 15, weight: .regular, design: .default))
                 }.padding(.horizontal)
+                 */
+                HStack(alignment: .center, spacing: 15) {
+                    // Ecosia: Disable accessibility label otherwise as it requires a mojor work on Firefox code to support it appropriately
```

Comment:
```suggestion
                    // Ecosia: Disable accessibility label otherwise as it requires a major work on Firefox code to support it appropriately
```